### PR TITLE
Remove govuk_frontend_toolkit from smart-answers

### PR DESCRIPTION
**Pull request by andysellick**
Remove `govuk_frontend_toolkit` from smart answers.

This has involved some structural changes (to replace the grid) some scss changes (to update variables and mixins) and some removal of unnecessary styles. This has resulted in some visual changes but I think they are within acceptable limits and bring the look of smart answers even more in line with govuk-frontend, which is a good thing.

Removal of classes did result in some test failures, these have been updated or made less specific as appropriate.

## Visual changes

Question intro before:

Screen Shot 2019-10-07 at 15 27 24

Question intro after (mostly the same):

Screen Shot 2019-10-07 at 15 27 31

Question before:

Screen Shot 2019-10-07 at 15 25 57

Question after (the big change is the previously answered questions, which is now using a govuk-frontend table. Since the font is bigger it didn't seem sensible to constrict it to 2/3rds width anymore, and there was no obvious place to put the 'start again' link, so I've just put it beneath the caption for simplicity):

Screen Shot 2019-10-07 at 15 27 04

Result before:

Screen Shot 2019-10-07 at 15 25 12

Result after:

Screen Shot 2019-10-07 at 17 33 07

### CODE REVIEW

**Comment by andysellick** [comment]
Quick test of your observation skills... this PR has managed to remove the breadcrumb, and I don't know why. It only appeared in the question intro page, and only seems to ever show 'Home' but I don't like that it disappeared. 

I didn't knowingly remove it, can anyone shed any light on this?

**Comment by vanitabarrett** [comment]
@andysellick Looks like the breadcrumb just doesn't work on Heroku

File: app/assets/stylesheets/smart_answers.scss [hunk 1]
```diff
@@ -1,20 +1,5 @@
...
-    } @else if $size == tablet {
-      @media (max-width: 800px){
-        @content;
-      }
-    }
-  }
-}
 
 // Styles
 // The more generic SmartAnswer CSS is in Static _multi_step.scss
```
[the remark this answers is not in the document]
Comment (andysellick) [reply app/assets/stylesheets/smart_answers.scss]: My next move. Well, when I find time 😁

File: app/views/smart_answers/_previous_answers.html.erb [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-            <% end %>
-          </tbody>
-        </table>
-      </article>
-    </div>
-  </div>
+  <table class="govuk-table govuk-!-margin-top-9">
+    <caption class="govuk-table__caption">
+      Previous answers
+      <p><%= link_to "Start again", smart_answer_path(params[:id]), :class => "govuk-link" %></p>
```
[the remark this answers is not in the document]
Comment (andysellick) [reply app/views/smart_answers/_previous_answers.html.erb]: None of the other text appears to be. I think that's outside the scope of this PR.

**Comment by andysellick** [comment]
@vanitabarrett I think I've addressed your comments. The weird hover/focus thing with the start button was due to an underly-specific rule to apply the focus state to links in the body text, fixed now.

**Pull request merged**